src/guide/introduction.md

```diff
@@ -189,7 +189,7 @@ There are quite a few other directives, each with its own special functionality.
 ```html
 <div id="list-rendering">
   <ol>
-    <li v-for="todo in todos" v-bind:key="todo.text">
```

Comment:
@skirtles-code after some thinking I agreed with your point and removed this attribute to prevent a cognitive overhad